package.json

```diff
@@ -9,7 +9,7 @@
     "url": "http://git-wip-us.apache.org/repos/asf/cordova-js.git"
   },
   "engines": {
-    "node": "~0.10.x"
+    "node": ">=0.10.x"
```

Comment:
This seems fine and should be a non-issue for most people - but I think we should just remove it unless we really have a dependency on node >= 0.10